src/controllers/AdminController.ts

```diff
@@ -139,7 +139,7 @@ export class AdminAPI extends Controller {
    * @param environmentId The environment id
    * @param body          The template resource to create
    */
-  @Post("project/{projectId}/templates")
+  @Post("project/{projectId}/template")
```

Comment:
This will break the current API

Reply:
@oliverqx  Here's one option to handle this without breaking the current API:-
- Keep the API route as is (`/templates`)
- If the body is an array then we know it's an array of templates being added, otherwise it is just one template

Reply:
@oliverqx you can revert this change and perform the following changes to be consistent.